packages/ariakit/src/checkbox/__examples__/checkbox-group/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Checkbox, useCheckboxState } from "ariakit/checkbox";
+import "./style.css";
+
+export default function Example() {
+  const checkbox = useCheckboxState({
+    defaultValue: [],
+  });
+
+  return (
+    <div>
+      <label className="label">
+        <Checkbox className="checkbox" state={checkbox} value="apple" /> Apple
+      </label>
+      <label className="label">
+        <Checkbox className="checkbox" state={checkbox} value="orange" /> Orange
+      </label>
+      <label className="label">
+        <Checkbox className="checkbox" state={checkbox} value="mango" /> Mango
+      </label>
+    </div>
+  );
+}
```

Comment:
We should probably use `Group` + `GroupLabel` here. Something like this:
```suggestion
import { Checkbox, useCheckboxState } from "ariakit/checkbox";
import { Group, GroupLabel } from "ariakit/group";
import "./style.css";

export default function Example() {
  const checkbox = useCheckboxState({ defaultValue: [] });
  return (
    <Group className="group">
      <GroupLabel className="group-label">Your favorite fruits</GroupLabel>
      <label className="label">
        <Checkbox className="checkbox" state={checkbox} value="apple" /> Apple
      </label>
      <label className="label">
        <Checkbox className="checkbox" state={checkbox} value="orange" /> Orange
      </label>
      <label className="label">
        <Checkbox className="checkbox" state={checkbox} value="mango" /> Mango
      </label>
    </Group>
  );
}
```